test/Removal.test.ts

```diff
@@ -186,4 +226,44 @@ describe('Removal', () => {
       });
     });
   });
+  describe('getters', () => {
+    describe('getScheduleStartTimeForRemoval', () => {
+      it('should return the restriction schedule start time for a removal id', async () => {
+        const { fifoMarket, removal, hre } = await setupTest();
+        const removalBalances = [100].map((balance) =>
+          formatTokenAmount(balance)
+        );
```

Comment:
the various `[singleElement].map` usages in this test seem unnecessary

Reply:
looks like it was just this one, fixed